README.md

```diff
@@ -7,188 +7,86 @@ apuX platform users and developers to customize firmware to their needs.
-    cd coreboot-${BR_NAME}/
-    cp /xgcc/.xcompile . # this command setup toolchain
-    cp configs/pcengines_apu2.config .config
-    make -j$(nproc)
-    ```
-
-5. once done find the new firmware in the build folder (`build/coreboot.rom`)
-6. see [here](http://pcengines.ch/howto.htm#bios) for instructions on how to
-   flash the firmware
-
-Building iPXE
--------------
-
-```sh
-cd && cd coreboot-${BR_NAME}
-IPXE_PATH=payloads/external/ipxe
-git clone https://github.com/pcengines/ipxe $IPXE_PATH
-wget https://raw.githubusercontent.com/pcengines/apu2-documentation/master/ipxe/general.h -O $IPXE_PATH/src/config/local/general.h
-wget https://raw.githubusercontent.com/pcengines/apu2-documentation/master/ipxe/menu.ipxe -O $IPXE_PATH/src/menu.ipxe
-cd $IPXE_PATH/src
-make -j$(nproc) bin/8086157b.rom EMBED=./menu.ipxe
-```
+Choose `Debian stable netboot 4.14.y` after boot login (`[root:debian]`) and
+run:
 
-Feel free to customize `menu.pxe` and `local/general.h` to match your needs.
-
-Building sortbootorder
-----------------------
-
-> coreboot is in `./coreboot-${BR_NAME}` directory
->
-```sh
-git clone https://github.com/pcengines/sortbootorder.git sortbootorder
-cd sortbootorder
-# for mainline coreboot (4.5.x)
-COREBOOT_ROOT=../coreboot-${BR_NAME} make distclean
-COREBOOT_ROOT=../coreboot-${BR_NAME} make
-# for legacy coreboot (4.0.x)
-COREBOOT_ROOT=../coreboot-${BR_NAME} make distclean
-COREBOOT_ROOT=../coreboot-${BR_NAME} COREBOOT_REL=legacy make
 ```
-
-Building memtest86+
--------------------
-
-```sh
-cd && cd coreboot-${BR_NAME}
-git clone https://github.com/pcengines/memtest86plus.git payloads/external/memtest86plus
-cd payloads/external/memtest86plus
-make -j$(nproc)
+flashrom -p internal -w apuX_x.y.z.rom
 ```
 
-cbfstool and adding/removing ROMs or payloads
----------------------------------------------
-
-`cbfstool` is result of `coreboot` build it gives ability to manipulate CBFS
-which filesystem for coreboot ROM.
-
-Usage examples:
-
-### Add iPXE ROM
+## Known issues
 
-```sh
-cd && cd coreboot-${BR_NAME}
-./build/cbfstool ./build/coreboot.rom add -f payloads/external/ipxe/src/bin/8086157b.rom -n genroms/pxe.rom -t raw
-```
+### Board mismatch
 
-Above command add raw image (`8086157b.rom`, result of iPXE build) to
-`coreboot.rom` under the name `genroms/pxe.rom`. This makes SeaBIOS to auto
-detect iPXE ROM and execute it before entering menu.
+Some binaries may need `boardmismatch=force` flashrom option because of SMBIOS
+table issue we had in old releases. Please double check you flashing correct
+binary before forcing.
 
-### Add sortbootorder
+### Incorrect privileges
 
 ```
-# add sortbootorder
-cd && cd coreboot-${BR_NAME}
-./build/cbfstool ./build/coreboot.rom remove -n img/setup
-./build/cbfstool ./build/coreboot.rom add-payload -f payloads/pcengines/sortbootorder/sortbootorder.elf -n img/setup -t payload
+$ ./start.sh
+Error checking context: 'can't stat '/path/pxe-server/debian/debian-stable/var/lib/sudo''.         '
```

Comment:
I have a fix for that. I will link a PR here in a moment.

Reply:
@miczyg1  still waiting for that

Reply:
https://github.com/3mdeb/pxe-server/pull/23